src/Components/Input.tsx

```diff
@@ -1,29 +1,184 @@
 import * as React from "react";
-import { View, Text, TextInput, StyleSheet } from "react-native";
+import { StyleSheet, Text, TextInput, Animated } from "react-native";
+import Colors from "../Themes/Colors";
+import { isWeb } from "../Helpers/platform";
 
-// DEMO VERSION
-type InputProps = {
-  field: string;
-  value: string;
-  onChangeText: (value: string) => void;
+type Props = {
```

Comment:
Maybe we can just extend native Input' props? (just a thought)

Reply:
extended